### PR TITLE
Added cards & Labels (needed for 18359)

### PR DESCRIPTION
**Pull request by nickmessing**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://nickmessing-features-18359-foundation-modules-for-plans.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://nickmessing-features-18359-foundation-modules-for-plans.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/assets/stylesheets/_settings.scss [hunk 1]
```diff
@@ -324,15 +324,15 @@ $button-radius: 2px;
...
-// $card-background: #fff;
-// $card-color: isitlight($card-background);
-// $card-border: 1px solid smartscale($card-background, 7%);
-// $card-radius: $global-radius;
-// $card-shadow: 0 1px 2px rgba(#000, 0.2);
-// $card-padding: $global-padding;
-// $card-margin: 0.5rem;
+$card-background: #fff;
+$card-color: isitlight($card-background);
+$card-border: 1px solid ;
```
Comment (floriank) [inline app/assets/stylesheets/_settings.scss]: I would very much prefer you put the settings for our `card component` into it's own file. same goes for labels - I did it for the `tooltip` component already.

Comment (myabc) [inline app/assets/stylesheets/_settings.scss]: :+1: for @floriank's suggestion: 

add a `_cards.md` and `_cards.s(c|a)ss` partial.

Comment (floriank) [inline app/assets/stylesheets/_settings.scss]: same probably for `labels`.

And I'd suggest using `.sass` :wink:

Comment (myabc) [inline app/assets/stylesheets/_settings.scss]: https://github.com/sass/sass/issues/1088 is a big annoyance, so I'd suggest `.scss` (see discussion here: https://community.openproject.org/work_packages/18422) :stuck_out_tongue:

**Comment by myabc** [comment]
@nickmessing I think we may have given you the wrong advice here :(

I'd only suggest moving the Foundation variables into partials, if you're using the mixins rather than the generated Foundation classes.

**Comment by myabc** [comment]
@nickmessing can you squash this down to one commit?

**Comment by myabc** [comment]
Let's squash this + merge to `release/4.1`.

**Comment by nickmessing** [comment]
@myabc squashed

**Comment by myabc** [comment]
Merged into `release/4.1` as 878e786e.

**Pull request closed without merging**